src/pow.cpp

```diff
@@ -89,6 +89,19 @@ unsigned int static DarkGravityWave(const CBlockIndex* pindexLast, const Consens
         return bnPowLimit.GetCompact();
     }
 
+    if (params.fPowAllowMinDifficultyBlocks && (
+        // testnet ...
+        (params.hashDevnetGenesisBlock.IsNull() && pindexLast->nChainWork >= UintToArith256(uint256S("0x000000000000000000000000000000000000000000000000003e9ccfe0e03e01"))) ||
```

Comment:
Why is this needed when `fPowAllowMinDifficultyBlocks` is set per network?

Reply:
Because otherwise almost the whole testnet chain is invalid - bits won't match

Reply:
Ok, just read the description and understand why this is here. Might be helpful to put in `// TEMP condition + description` comment in the code so the seemingly "random" target value makes sense.